spec-tests/src/test/java/org/candlepin/spec/imports/ImportUpdateSpecTest.java

```diff
@@ -0,0 +1,173 @@
+            exportGenerator.getContent().getId());
+        assertNotNull(originalContent);
+
+        Export updatedExport = exportGenerator.updateWithProduct().export();
+        doImport(importOwner.getKey(), updatedExport.file());
+
+        assertThat(subs).hasSize(1);
+        List<SubscriptionDTO> newSubs = admin.owners().getOwnerSubscriptions(importOwner.getKey());
+        assertThat(newSubs).hasSize(3);
+
+        // ensure that the original sub doesn't have the same serial id
+        SubscriptionDTO oldSub = subs.get(0);
+        boolean found = false;
+        List<SubscriptionDTO> matches = newSubs.stream()
+            .filter(x -> x.getId().equals(oldSub.getId()))
+            .collect(Collectors.toList());
+        if (matches != null && matches.size() > 0) {
+            assertNotEquals(matches.get(0).getCertificate().getSerial().getId(),
+                oldSub.getCertificate().getSerial().getId());
+            found = true;
+        }
+        // make sure that the original sub was in the new list
+        assertTrue(found);
+
+        ProductDTO updatedProduct = ownerProductApi.getProductByOwner(importOwner.getKey(),
+            exportGenerator.getProduct().getId());
+        ContentDTO updatedContent = ownerContentApi.getOwnerContent(importOwner.getKey(),
+            exportGenerator.getContent().getId());
+        assertNotNull(updatedProduct);
+        assertNotNull(updatedContent);
+        assertNotEquals(updatedProduct, originalProduct);
+        assertNotEquals(updatedContent, originalContent);
+    }
+
+    @Test
+    public void shouldRemoveAllImportedSubscriptionsIfImportHasNoEntitlements() throws Exception {
+        ProductDTO product = Products.randomEng();
+        ExportGenerator exportGenerator = new ExportGenerator(admin);
+        Export export = exportGenerator.minimal().withProduct(product).export();
+
+        OwnerDTO importOwner = Owners.random();
+        importOwner = admin.owners().createOwner(importOwner);
+        UserDTO importUser = UserUtil.createUser(admin, importOwner);
+        ApiClient importUserClient = ApiClients.basic(importUser);
+        doImport(importOwner.getKey(), export.file());
+
+        //  The manifest metadata can end up with a created date that is a fraction of a second ahead of
+        //  the created date in the cp_export_metadata table. This results into the manifest metadata
+        //  conflict with error "Import is the same as existing data". Hence to avoid this, adding a sleep
+        //  before creating another export.
+        Thread.sleep(2000);
+        Export updatedExport = exportGenerator.updateWithProductNoEnt().export();
+        doImport(importOwner.getKey(), updatedExport.file());
+
+        assertThat(exportGenerator.getManifestConsumerEntitlements()).hasSize(0);
+        assertThat(admin.owners().getOwnerSubscriptions(importOwner.getKey())).hasSize(0);
+    }
+
+    private void doImport(String ownerKey, File export) {
```

Comment:
Why are we abstracting away a one-liner? If this is done commonly enough that the `List.of()` is becoming burdensome, we should be adding an overload of `importManifestAsync` which takes only the ownerKey and export file instead of this.